tests/create_LWE.py

```diff
@@ -0,0 +1,322 @@
+    lrmp = LumpedRateModelWithPores(
+        component_system, name='lrmp')
+
+    lrmp.length = 0.014
+    lrmp.diameter = 0.01 * 2
+    lrmp.bed_porosity = 0.37
+    lrmp.axial_dispersion = 5.75e-8
+
+    configure_discretization(lrmp, **kwargs)
+    configure_particles(lrmp, **kwargs)
+    configure_adsorption(lrmp, component_system, **kwargs)
+    configure_film_diffusion(lrmp, component_system.n_comp)
+    configure_flow_direction(lrmp, **kwargs)
+
+    return lrmp
+
+
+def configure_mct(component_system: ComponentSystem, **kwargs) -> MCT:
+    mct = MCT(component_system, nchannel=3, name='mct')
+
+    mct.length = 0.014
+    mct.channel_cross_section_areas = 3 * [2 * np.pi * (0.01 ** 2)]
+    mct.axial_dispersion = 5.75e-8
+
+    n_comp: int = component_system.n_comp
+
+    mct.exchange_matrix = np.array([
+        [n_comp * [0.0], n_comp * [0.001], n_comp * [0.0]],
+        [n_comp * [0.002], n_comp * [0.0], n_comp * [0.003]],
+        [n_comp * [0.0], n_comp * [0.0], n_comp * [0.0]]
+    ])
+
+    configure_discretization(mct, **kwargs)
+    configure_flow_direction(mct, **kwargs)
+
+    return mct
+
+
+def configure_discretization(unit_operation, **kwargs) -> None:
+    n_col: int = kwargs.get('n_col', 100)
+    n_par: int = kwargs.get('n_par', 1)
+    ad_jacobian: bool = kwargs.get('ad_jacobian', False)
+
+    if 'npar' in unit_operation.discretization.parameters:
+        unit_operation.discretization.npar = n_par
+        unit_operation.discretization.par_disc_type = 'EQUIDISTANT_PAR'
+
+    unit_operation.discretization.ncol = n_col
+    unit_operation.discretization.use_analytic_jacobian = not ad_jacobian
+
+
+def configure_particles(unit_operation) -> None:
+    par_radius = 4.5e-5
+    par_porosity = 0.75
+
+    unit_operation.particle_radius = par_radius
+    unit_operation.particle_porosity = par_porosity
+    unit_operation.discretization.par_geom = 'SPHERE'
+
+
+def get_binding_model_class(binding_type: str, component_system: ComponentSystem):
+    binding_classes = {
+        'linear': Linear,
+        'langmuir': Langmuir,
+        'langmuir_ldf': LangmuirLDF,
+        'bilangmuir': BiLangmuir,
+        'bilangmuir_ldf': BiLangmuirLDF,
+        'freundlich_ldf': FreundlichLDF,
+        'steric_mass_action': StericMassAction,
+        'anti_langmuir': AntiLangmuir,
+        'spreading': Spreading,
+        'self_association': SelfAssociation,
+        'bi_steric_mass_action': BiStericMassAction,
+        'multistate_steric_mass_action': MultistateStericMassAction,
+        'simplified_multistate_steric_mass_action': SimplifiedMultistateStericMassAction,
+        'saska': Saska,
+        'generalized_ion_exchange': GeneralizedIonExchange
+    }
+
+    try:
+        return binding_classes[binding_type](component_system)
+    except KeyError:
+        raise ValueError(f'Unknown binding type {binding_type}')
+
+
+def configure_adsorption(unit_operation, component_system, **kwargs):
```

Comment:
Can do that but then would break with the convention of naming similar to the LWE in cadet core 🤔 
Maybe instead I could set steric_mass_action to default for now (in the function call)?

Reply:
jein, in CADET-Core we also have `createMCLin`, `createSCLang`, `createSCLin`, and `createSCLinStep` functions which use single or multi component linear or Langmuir isotherms